projects/step-core/src/lib/modules/custom-forms/components/custom-form-input/base-custom-form-input.component.ts

```diff
@@ -32,6 +32,12 @@ export abstract class BaseCustomFormInputComponent implements ControlValueAccess
   /** @Output **/
   readonly touch = output();
 
+  ngAfterViewInit(): void {
+    if (this.input()?.type === InputType.CHECKBOX) {
```

Comment:
@neogucky Please, undo this commit. Form boolean values are initialized in other place.
Form control should not contain any logic, that affects the model, except user's manual change